### PR TITLE
fix(web): Use <a> tags for links that are pointing to a project path (like /minarsidur)

### PR DESCRIPTION
**Pull request by RunarVestmann**
# Use `<a>` tags for links that are pointing to a project path (like /minarsidur)

## What

* When users click on a link to /samradsgatt on the web it'll show them an error page instead of redirecting them to the consultation portal since we're at the moment using NextLinks

There are two downsides to this approach
1. We need to maintain the `ProjectBasePath` enum in the code and make sure we update it when new projects are added
2. This might mean that sometimes we'll create `<a>` tags when they technically aren't necessary (this can be the case if a project has a base path that uses the Link component

## Why

* We want links to work correctly in next.js when they point to another hosted project

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/shared/utils/src/lib/shouldLinkBeAnAnchorTag.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+/** Some links point users to another project (like if a user clicks on a /minarsidur link in the web project),
+ * then the link should be an anchor tag instead of a nextlink for example
+ * */
+export const shouldLinkBeAnAnchorTag = (path: string) => {
+  for (const basePath of Object.values(ProjectBasePath)) {
+    if (path.includes(basePath)) return true
+  }
+  return false
+}
```
Comment (saevarma) [inline libs/shared/utils/src/lib/shouldLinkBeAnAnchorTag.ts:11]: ?
```suggestion
export const shouldLinkBeAnAnchorTag = (path: string) => Object.values(ProjectBasePath).includes(path)
```

Comment (RunarVestmann) [reply libs/shared/utils/src/lib/shouldLinkBeAnAnchorTag.ts:11]: Let's say the path is "/minarsidur/eitthvad"

If that gets passed in it'll return true with the previous version of the function but if we use your version of the function it'll return false.

Comment (saevarma) [reply libs/shared/utils/src/lib/shouldLinkBeAnAnchorTag.ts:11]: Yup agreed. You version handles nested routes which we need to do.

File: libs/shared/constants/src/lib/projectBasePath.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export enum ProjectBasePath {
+  ServicePortal = '/minarsidur',
+  ConsultationPortal = '/samradsgatt',
+  ApplicationSystem = '/umsoknir',
+  AirDiscountScheme = '/loftbru',
```
Comment (saevarma) [inline libs/shared/constants/src/lib/projectBasePath.ts:5]: AirDiscountScheme has been moved so `/loftbru` is actually a page on the web and the application link is now `/minarsidur/loftbru`. So we should drop this so we don't render `<a>` tag for a internal link.

File: libs/shared/constants/src/lib/projectBasePath.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  AdminPortal = '/stjornbord',
+}
```
Comment (saevarma) [inline libs/shared/constants/src/lib/projectBasePath.ts:7]: We should add `CarRecycling = '/app/skilavottord`` for skilavottord-web

**Review by saevarma — APPROVED** [review]
Just minor changes on the paths otherwise looks good to me.

File: libs/shared/constants/src/lib/projectBasePath.ts (hunk 3 above)
Comment (RunarVestmann) [reply libs/shared/constants/src/lib/projectBasePath.ts:7]: Done

**Review by thorkellmani — APPROVED** [review]
LGTM

**Pull request merged**